Makefile

```diff
@@ -396,13 +397,143 @@ test\#%:
+		code.gitea.io/gitea/modules/translation=60.70% \
+		code.gitea.io/gitea/modules/typesniffer=92.86% \
+		code.gitea.io/gitea/modules/updatechecker=0.00% \
+		code.gitea.io/gitea/modules/upload=73.85% \
+		code.gitea.io/gitea/modules/uri=27.78% \
+		code.gitea.io/gitea/modules/user=23.53% \
+		code.gitea.io/gitea/modules/util=72.93% \
+		code.gitea.io/gitea/modules/validation=79.4% \
+		code.gitea.io/gitea/modules/watcher=0.00% \
+		code.gitea.io/gitea/modules/web=76.76% \
+		code.gitea.io/gitea/modules/webhook=13.3% \
+		code.gitea.io/gitea/models/activities=58.48% \
+		code.gitea.io/gitea/models/admin=33.55% \
+		code.gitea.io/gitea/models/asymkey=41.40% \
+		code.gitea.io/gitea/models/auth=46.36% \
+		code.gitea.io/gitea/models/avatars=31.82% \
+		code.gitea.io/gitea/models/db=41.53% \
+		code.gitea.io/gitea/models/git=44.19% \
+		code.gitea.io/gitea/models/issues=57.31% \
+		code.gitea.io/gitea/models/organization=71.00% \
+		code.gitea.io/gitea/models/packages=71.66% \
+		code.gitea.io/gitea/models/perm=57.05% \
+		code.gitea.io/gitea/models/project=31.85% \
+		code.gitea.io/gitea/models/pull=36.13% \
+		code.gitea.io/gitea/models/repo=58.62% \
+		code.gitea.io/gitea/models/secret=24.56% \
+		code.gitea.io/gitea/models/system=56.16% \
+		code.gitea.io/gitea/models/unit=48.72% \
+		code.gitea.io/gitea/models/unittest=31.91% \
+		code.gitea.io/gitea/models/user=55.82% \
+		code.gitea.io/gitea/models/webhook=66.02% \
+		code.gitea.io/gitea/routers/api=60.66% \
+		code.gitea.io/gitea/routers/common=61.15% \
+		code.gitea.io/gitea/routers/install=5.7% \
+		code.gitea.io/gitea/routers/private=34.74% \
+		code.gitea.io/gitea/routers/utils=100.00% \
+		code.gitea.io/gitea/routers/web=28.95% \
+		code.gitea.io/gitea/services/agit=56.59% \
+		code.gitea.io/gitea/services/asymkey=36.73% \
+		code.gitea.io/gitea/services/attachment=60.87% \
+		code.gitea.io/gitea/services/auth=38.99% \
+		code.gitea.io/gitea/services/automerge=44.83% \
+		code.gitea.io/gitea/services/context=47.22% \
+		code.gitea.io/gitea/services/convert=73.5% \
+		code.gitea.io/gitea/services/cron=56.2% \
+		code.gitea.io/gitea/services/externalaccount=0.00% \
+		code.gitea.io/gitea/services/forms=36.5% \
+		code.gitea.io/gitea/services/gitdiff=70.52% \
+		code.gitea.io/gitea/services/issue=40.50% \
+		code.gitea.io/gitea/services/lfs=57.06% \
+		code.gitea.io/gitea/services/mailer=54.55% \
+		code.gitea.io/gitea/services/markup=100.00% \
+		code.gitea.io/gitea/services/migrations=35.97% \
+		code.gitea.io/gitea/services/mirror=29.83% \
+		code.gitea.io/gitea/services/org=49.02% \
+		code.gitea.io/gitea/services/packages=51.94% \
+		code.gitea.io/gitea/services/pull=43.34% \
+		code.gitea.io/gitea/services/release=45.09% \
+		code.gitea.io/gitea/services/repository=41.19% \
+		code.gitea.io/gitea/services/task=43.75% \
+		code.gitea.io/gitea/services/user=46.24% \
+		code.gitea.io/gitea/services/webhook=66.63% \
+		code.gitea.io/gitea/services/wiki=56.41% \
```

Comment:
It seems clumsy to have these statically in the Makefile. How about writing them out to a file, loading it from there and also provide a `make` target to update them to current coverage results? Ideally that update script could run in scheduled job weekly.